### PR TITLE
English comments were changed to French

### PR DESCRIPTION
**Pull request by ceokzoo**

### CODE REVIEW

**Comment by ficristo** [comment]
Usually French and Japanese translation are handled by Adobe.

**Comment by ceokzoo** [comment]
Oh okay. What are the next steps to deal with the translation?

**Comment by ficristo** [comment]
Patentially waiting someone at Adobe. Sorry but I cannot help more.

**Comment by swmitra** [comment]
@ceokzoo Sorry missed this thread. I can't review the actual strings but I am asking our localization team to have a look at it. Once they approve I will merge this PR.

File: samples/fr/Premiers pas/index.html [hunk 1]
```diff
@@ -14,23 +14,23 @@
         <h2>Suivez le guide !</h2>
         
         <!--
-            MADE WITH <3 AND JAVASCRIPT
+            FAIT AVEC <3 ET JAVASCRIPT
```
Comment (swmitra) [inline samples/fr/Premiers pas/index.html]: Can we change this to - 'CRÉÉ AVEC <3 ET JAVASCRIPT'

File: samples/fr/Premiers pas/index.html [hunk 2]
```diff
@@ -14,23 +14,23 @@
...
         -->
         
         <p>
             Bienvenue dans Brackets, un éditeur de code open source qui comprend et facilite la conception de sites web. Il s’agit d’un éditeur à la fois léger et puissant qui intègre des outils visuels directement dans son interface, de sorte que chaque opération peut devenir un véritable jeu d’enfant.
         </p>
         
         <!--
-            WHAT IS BRACKETS?
+            QU'EST-CE QUE <<BRACKETS>>?
         -->
```
Comment (swmitra) [inline samples/fr/Premiers pas/index.html]: QU’EST-CE QUE BRACKETS ? (missing non-breaking space before question mark)

File: samples/fr/Premiers pas/index.html [hunk 3]
```diff
@@ -14,23 +14,23 @@
...
+            FAIT AVEC <3 ET JAVASCRIPT
         -->
         
         <p>
             Bienvenue dans Brackets, un éditeur de code open source qui comprend et facilite la conception de sites web. Il s’agit d’un éditeur à la fois léger et puissant qui intègre des outils visuels directement dans son interface, de sorte que chaque opération peut devenir un véritable jeu d’enfant.
         </p>
         
         <!--
-            WHAT IS BRACKETS?
+            QU'EST-CE QUE <<BRACKETS>>?
```
Comment (swmitra) [inline samples/fr/Premiers pas/index.html]: QU’EST-CE QUE BRACKETS ? (missing non-breaking space before question mark)

File: samples/fr/Premiers pas/index.html [hunk 4]
```diff
@@ -14,23 +14,23 @@
         <h2>Suivez le guide !</h2>
         
         <!--
-            MADE WITH <3 AND JAVASCRIPT
+            FAIT AVEC <3 ET JAVASCRIPT
         -->
         
         <p>
             Bienvenue dans Brackets, un éditeur de code open source qui comprend et facilite la conception de sites web. Il s’agit d’un éditeur à la fois léger et puissant qui intègre des outils visuels directement dans son interface, de sorte que chaque opération peut devenir un véritable jeu d’enfant.
```
Comment (swmitra) [inline samples/fr/Premiers pas/index.html]: This paragraph may be edited like - 
‘’ Bienvenue dans Brackets, un éditeur de code open source qui comprend et facilite la conception de sites web. Il s’agit d’un éditeur à la fois léger et puissant qui intègre des outils visuels directement dans son interface, de sorte que chaque opération peut devenir un véritable jeu d’enfant.”

File: samples/fr/Premiers pas/index.html (hunk 1 above)
Comment (ceokzoo) [inline samples/fr/Premiers pas/index.html]: Yes!

**Comment by ceokzoo** [comment]
Updated the file with your suggestions!

**Review by swmitra — APPROVED** [review]

**Comment by swmitra** [comment]
Merging PR #12905

**Pull request merged**